### PR TITLE
[11.0][FIX] mgmtsystem_nonconformity: Various fixes

### PR DESCRIPTION
**Pull request by jarroyomorales**
This PR implements a couple of minor fixes to `mgmtsystem_nonconformity`.
- State is now not readonly and present in stage's form.
- Added sheet to some model's forms
- Generated readme using maintainer tools
- Other formatting fixes.

cc. @etobella

### CODE REVIEW

**Review by etobella — APPROVED** [review]
LGTM :+1:

File: mgmtsystem_nonconformity/readme/USAGE.rst [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+* Partner : Customer, supplier or internal personnel
+* Related to: Any reference pointing to the NC (order id, project id, task id, etc.)
+* Responsible: Person responsible for the NC
+* Manager : Person managing the department or owner of the procedure
+* Filled in by: Originator of NC report
+* Origins:  The source of the NC, how was it discover
+* Procedures:  Against which procedure is the NC
+* Description: Evidence, reference to the standards
+
+* Click on Save and then on Analysis.
```
Comment (mymage) [inline mgmtsystem_nonconformity/readme/USAGE.rst]: For me better:
* Click on Save button and then on "Analysis" on stage sequence in the top right corner.

Comment (mymage) [inline mgmtsystem_nonconformity/readme/USAGE.rst]: For me better:
`Click on Save button and then on "Analysis" on stage sequence in the top right corner.`

File: mgmtsystem_nonconformity/readme/USAGE.rst [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+Go to the newly created NC and fill in the following
+information in the tab named Causes and Analysis:
+
+* Causes: Add root causes
+* Analysis: Describe the results of the investigation
+* Severity: Select the severity among unfounded, minor and major
+* Immediate action: Create or select an immediate action if appropriate
+
+Click on the Save button and then on the "Action Plan" button in the top right corner.
```
Comment (mymage) [inline mgmtsystem_nonconformity/readme/USAGE.rst]: For me better:
`Click on the Save button and then on "Action Plan" on stage sequence in the top right corner.`

File: mgmtsystem_nonconformity/readme/USAGE.rst [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+* Responsible: Person in charge for implementing the action
+* Type: Immediate, corrective or preventive actions or improvement opportunity
+* Description: Details of the action
+
+When the action is created, a notification is sent to the person responsible
+for the action.
+
+Enter comments into the input field below the "Plan Review" section, those comments are required to reach the next stage.
+
+To begin the work on the planned Actions change the stage of the NC to open by clicking on the "In Progress" button in the top right corner.
```
Comment (mymage) [inline mgmtsystem_nonconformity/readme/USAGE.rst]: For me better:
`To begin the work on the planned Actions change the stage of the NC to open by clicking on "In Progress" on stage sequence in the top right corner.`

File: mgmtsystem_nonconformity/readme/USAGE.rst [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+When all actions of the plan are done, their effectiveness must be evaluated
+before closing the NC.
```
Comment (mymage) [inline mgmtsystem_nonconformity/readme/USAGE.rst]: For me better:
`before closing the NC clicking on the "Closed" on stage sequence in the top right corner.`

File: mgmtsystem_nonconformity/readme/DESCRIPTION.rst [hunk 5]
```diff
@@ -0,0 +1,7 @@
+This module enables you to manage the nonconformities of your management systems:
+
+* Quality (ISO 9001)
+* Environment (ISO 14001)
+* Information Security (ISO 27001)
+* Health and Safety (ISO 45001)
+* IT Services (ISO 20000)
```
Comment (mymage) [inline mgmtsystem_nonconformity/readme/DESCRIPTION.rst:7]: Add a:
`* Other custom management system`

**Review by mymage — CHANGES_REQUESTED** [review]
Be patien if I did some mistake, it's my first review ;-)

**Comment by jarroyomorales** [comment]
@mymage Thanks for the review! I just added your changes!

**Review by mymage — APPROVED** [review]

**Review by max3903 — APPROVED** [review]

**Pull request merged**